libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3469,6 +3469,9 @@ MAV_RESULT GCS_MAVLINK::handle_preflight_reboot(const mavlink_command_int_t &pac
             send_text(MAV_SEVERITY_WARNING,"deadlock passed");
             return MAV_RESULT_ACCEPTED;
         }
+        if (is_equal(packet.param4, 101.0f)) {
+            AP_BoardConfig::config_error("config_error - YOU~RE WELCOME!");  // the capital-U and ~ here are actually important for testing a MissionPlanner bug!
```

Comment:
The `config_error` method already prepends "Config Error: " to the message, so I guess let's save the 15 bytes
```suggestion
            AP_BoardConfig::config_error("YOU~RE WELCOME!");  // the capital-U and ~ here are actually important for testing a MissionPlanner bug!
```

```
2/2/2025 12:35:36 AM : Config Error: fix problem then reboot
2/2/2025 12:35:36 AM : Config Error: YOU~RE WELCOME!
```

Reply:
Done